src/hotspot/share/cds/filemap.cpp

```diff
@@ -2465,6 +2504,14 @@ bool FileMapHeader::validate() {
     return false;
   }
 
+  if (compact_headers() != UseCompactObjectHeaders) {
```

Comment:
(Commenting here, but the comment applies to code a bit above) While debugging CDS, it would have been useful to print the value of UseCompactObjectHeaders.

Could we change the code to be:
```
  log_info(cds)("Archive was created with UseCompressedOops = %d, UseCompressedClassPointers = %d, UseCompactObjectHeaders = %d",
                          compressed_oops(), compressed_class_pointers(), compact_headers());
```

Reply:
Resolved.